src/operator/nn/batch_norm.cc

```diff
@@ -28,7 +28,7 @@
 #include <nnvm/op_attr_types.h>
 #include "../elemwise_op_common.h"
 #include "../operator_common.h"
-#if MXNET_USE_MKLDNN == 1
+#if MXNET_USE_MKLDNN == 100
```

Comment:
What does 100 mean?

Reply:
As @TaoLv said: Here we simply use 100 and 1 to distinguish the code for MKL-DNN v0.x and v1.x so we don't need to remove v0.x integration code and hence convenient to review. When merging this branch back to master, we will change all MXNET_USE_MKLDNN == 100 back to MXNET_USE_MKLDNN == 1.

Reply:
Mentioned here https://github.com/apache/incubator-mxnet/pull/16141#discussion_r324531557

Reply:
Thanks for your review. Please see the discussion here: https://github.com/apache/incubator-mxnet/pull/15706#discussion_r311613093.